File: src/main/java/inraito/openerg/common/item/MESwitchingCardItem.java
package inraito.openerg.common.item;

import inraito.openerg.common.item.group.ModGroup;
import net.minecraft.item.Item;

class MESwitchingCardItem extends Item{
    public MESwitchingCardItem() {
        super(new Properties().tab(ModGroup.openEnergisticsGroup));
    }
}

File: src/main/scala/inraito/openerg/common/item/MESwitchingCardItem.scala
package inraito.openerg.common.item

import inraito.openerg.common.item.group.ModGroup
import li.cil.oc.common.item.traits.SimpleItem
import net.minecraft.item.Item
import net.minecraft.item.Item.Properties
import net.minecraftforge.common.extensions.IForgeItem

class MESwitchingCardItem extends Item(new Properties().tab(ModGroup.openEnergisticsGroup)) with SimpleItem with IForgeItem{
  override def getDescriptionId(): String = super.getOrCreateDescriptionId();

}
